ironfish/src/workerPool/pool.ts

```diff
@@ -225,4 +228,48 @@ export class WorkerPool {
 
     return response.verified
   }
+
+  async boxMessage(
```

Comment:
I'm not super familiar with the worker pool tests, but do you think we could add some test cases for these methods?

Reply:
The methods are actually tested via the peer network which calls these functions already